source/request-as-event-emitter.js

```diff
@@ -122,8 +120,9 @@ module.exports = (options = {}) => {
 
 		cacheReq.once('request', req => {
 			let aborted = false;
-			req.once('abort', _ => {
+			req.once('abort', () => {
```

Comment:
Refactoring should be done in a separate PR. Just Kidding :P 👍

Reply:
I considered it, but when I looked at the diff, I didn't think the one line change raised the cognitive burden enough to warrant a separate commit. 😅